src/types/rule.ts

```diff
@@ -62,6 +62,11 @@ export interface ILengthRuleOptions {
 }
 export type LengthRule = IRule<RuleFunction.LENGTH, ILengthRuleOptions>;
 
+export interface IEnumRuleOptions {
+  values: Array<'string' | number>;
```

Comment:
Shouldn't this be `string`?
```suggestion
  values: Array<string | number>;
```

Reply:
good point!